src/components/IPScan/Summary/index.js

```diff
@@ -329,6 +330,17 @@ const SummaryIPScanJob = ({
 
   const goTerms = getGoTerms(payload.matches);
 
+  let dataURL = 'https://www.ebi.ac.uk/Tools/services/rest/iprscan5/result/';
+  const now = Date.now();
+  const expired =
```

Comment:
Instead of using `now` as the failover of `created` maybe use something similar to [line 314](https://github.com/ProteinsWebTeam/interpro7-client/blob/4228c4fb5d15f1deb7987dc5106be18cc14f82b7/src/components/IPScan/Summary/index.js#L314) where the creation date is infer from the ipscan ID.

Reply:
or not sure if we can actually do that calculation when the job is been imported, just to make sure the `created` value is always there

Reply:
When it is imported 'created' value is undefined. But the whole things needn't be calculated for imported jobs as we are disabling the export anyway.

Reply:
but it's exactly at the point of deciding it should be disabled or not that it's important to have it available. 
In the current setup the 7 days are counted from the moment it's imported, but the job could have been imported 6 days after it was created, so it actually has only 1 day more in the server.